src/conn.rs

```diff
@@ -132,21 +133,33 @@ impl<T: EventMessage + Unpin> Stream for Connection<T> {
                     pin.pending_flush = Some(call);
                 }
             }
+
             break;
         }
 
         // read from the ws
         match ready!(pin.ws.poll_next_unpin(cx)) {
-            Some(Ok(msg)) => match serde_json::from_slice::<Message<T>>(&msg.into_data()) {
-                Ok(msg) => {
-                    tracing::trace!("Received {:?}", msg);
-                    Poll::Ready(Some(Ok(msg)))
-                }
-                Err(err) => {
-                    tracing::error!("Failed to deserialize WS response {}", err);
-                    Poll::Ready(Some(Err(err.into())))
-                }
-            },
+            Some(Ok(WsMessage::Text(text))) => {
+                let ready = match text.parse::<Message<T>>() {
+                    Ok(msg) => {
+                        tracing::trace!("Received {:?}", msg);
+                        Ok(msg)
+                    }
+                    Err(err) => {
+                        tracing::debug!(target: "chromiumoxide::conn::raw_ws::parse_errors", msg = text, "Failed to parse raw WS message");
```

Comment:
```suggestion
                        tracing::trace!(target: "chromiumoxide::conn::raw_ws::parse_errors", msg = text, "Failed to parse raw WS message");
```

This can potentially be a large steam of data, let's downgrade the log level here?